build.gradle

```diff
@@ -42,24 +42,31 @@ artifactory {
 
 jenkinsPlugin {
     pluginId = 'synopsys-security-scan-plugin'
-    jenkinsVersion = '2.387.3'
+    jenkinsVersion = '2.377'
     displayName = 'Synopsys Security Scan Plugin'
 }
 
 dependencies {
+    implementation 'org.jenkins-ci.plugins:cloudbees-bitbucket-branch-source:805.v7f97d29dc0f5'
+    implementation 'org.jenkins-ci.plugins:matrix-project:785.v06b_7f47b_c631'
+    implementation 'org.jenkins-ci.plugins:cloudbees-folder:6.15'
     annotationProcessor 'com.synopsys.integration:jenkins-annotation-processor:0.0.7'
 
+    implementation 'org.jenkins-ci.plugins:credentials:1139.veb_9579fca_33b_'
+    implementation 'com.synopsys.integration:blackduck-common:66.1.4'
```

Comment:
We may not need this for now, currently we are not using anything from any of these dependencies

Reply:
Blackduck-common is removed